markdown/events/2021/bytesize-16-module-test-data.md

```diff
@@ -27,3 +28,40 @@ This week, Kevin Menden ([@KevinMenden](http://github.com/KevinMenden/)) will pr
+
+[7:54](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=474) So how do you use that data when you write a test? The first thing you need to do is to identify the file you would need. To do that, as I previously mentioned, you need to visit the test dataset repository (go through the README) and take a look if the file is available, double-check where the file is located, and finally use that information to look for corresponding keys in the `test_data.config` file. You can use those keys to link the file in your test code. At that point, you should really be able to just run your test and finish your test code. That’s it then!
+
+[8:32](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=512) Unless of course you can’t find the data that’s necessary for your modules. So of course we don’t have all kinds of datasets online and there are all kinds of bioinformatics tools available. Some of them may need specific data to run the modules. So it’s quite possible that you will find that the data you want is not available. This isn’t a problem, you can just upload the data yourself (this will also be helpful for us!).
+
+[8:57](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=537) So first, it is important to read the instructions (in the `README` in the test dataset repository). We have a few lines about how to add new test data. Make sure to check this out and then think about a suitable minimal test dataset. The optimal thing here would be if you could find something that went with sarscov2 because that’s currently our favourite genome because it is small and there’s a lot of data available. So hopefully you can use this because then everything stays small and we don’t need to add a new genome. Then once you identify something, it is best to ask for opinions to gauge if it suitable for the test dataset repository in the `#modules` channel on Slack. Once you’ve established that it’s OK, you can create a pull request to the `modules` branch of the test dataset repository and ask for reviews `#modules` so that you can eventually have your pull request merged. Once that’s done and your data is available, you need to add an entry to the `test_data.config` file in the `modules` repository. That’s again a separate pull request to have an entry merge, but it shouldn’t be a problem.
```

Comment:
```suggestion
[8:57](https://youtu.be/QXfAerydAT0?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=537) So first, it is important to read the instructions (in the `README` in the test dataset repository). We have a few lines about how to add new test data. Make sure to check this out and then think about a suitable minimal test dataset. The optimal thing here would be if you could find something that went with sarscov2 because that’s currently our favourite genome because it is small and there’s a lot of data available. So hopefully you can use this because then everything stays small and we don’t need to add a new genome. Then once you identify something, it is best to ask for opinions to check if it's suitable for the test dataset repository in the `#modules` channel on Slack. Once you’ve established that it’s OK, you can create a pull request to the `modules` branch of the test dataset repository and ask for reviews `#modules` so that you can eventually have your pull request merged. Once that’s done and your data is available, you need to add an entry to the `test_data.config` file in the `modules` repository. That’s again a separate pull request to have an entry merge, but it shouldn’t be a problem.
```